tools/cold-backup/run.sh

```diff
@@ -117,9 +117,7 @@ __stop_cluster() {
 }
 
 __cleanup() {
-    if [[ $INCREMENTAL == false ]]; then
-        rm -rf $__TMP/$NAMESPACE/$CLUSTER_NAME
-    fi
+    rm -rf $BACKUP_PATH/$NAMESPACE/$CLUSTER_NAME
```

Comment:
Is this secure?

Reply:
Good point. We can replace all rm commands with an non-empty dir check.